Gemfile.lock

```diff
@@ -712,4 +759,4 @@ DEPENDENCIES
   websocket-client-simple!
 
 BUNDLED WITH
-   2.5.16
+   2.5.17
```

Comment:
There are a lot of changes in the Gemfile.lock, but your PR does not seem to require any; I would revert those